policyengine_us/variables/gov/states/ny/nyserda/drive_clean/ny_drive_clean_rebate.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class ny_drive_clean_rebate(Variable):
+    value_type = float
+    entity = Household
+    label = "New York Drive Clean Rebate"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.nyserda.ny.gov/-/media/Project/Nyserda/Files/Programs/Drive-Clean-NY/implementation-manual.pdf#page=8"
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ny.nyserda.drive_clean
+        # Calculate vehicle all-electric range
+        vehicle_range = tax_unit(
+            "ny_drive_clean_vehicle_electric_range", period
+        )
+        # Calculate vehicle price
+        vehicle_cost = tax_unit("ny_drive_clean_vehicle_cost", period)
```

Comment:
what are the criteria to qualify for this? can we generalize it to something like vehicle_purchase_price and other criteria, as used for the US clean vehicle tax credit?